static/app/views/starfish/views/spanSummaryPage/spanTransactionsTable.tsx

```diff
@@ -112,17 +113,22 @@ export function SpanTransactionsTable({
 
   return (
     <Fragment>
-      <GridEditable
-        isLoading={isLoading}
-        data={spanTransactionsWithMetrics}
-        columnOrder={COLUMN_ORDER}
-        columnSortBy={[]}
-        grid={{
-          renderHeadCell: col => renderHeadCell({column: col, sort, location}),
-          renderBodyCell,
-        }}
-        location={location}
-      />
+      <VisuallyCompleteWithData
+        id="SpanSummary.EndpointTable"
```

Comment:
I think this should be 
```suggestion
        id="SpanSummary.SpanTransactionsTable"
```

to match the component name